### PR TITLE
Smal fixes for https://github.com/Microsoft/visualfsharp/pull/5940

### PR DESCRIPTION
**Pull request by AviAvni**
Small fixes according to @blumu review

### CODE REVIEW

**Comment by cartermp** [comment]
@dsyme the code for this doesn't appear to have a test for correctness. Any ideas on adding a regression test for this?

File: src/fsharp/NameResolution.fs [hunk 1]
```diff
@@ -1541,12 +1541,11 @@ type TcResultsSinkImpl(g, ?source: string) =
...
-                            if c = '\r' && i + 1 < source.Length && source.[i+1] = '\n' then ()
-                            elif c = '\r' then yield i + 1
-                            if c = '\n' then yield i + 1
-                        yield source.Length
+                        for i in 1..source.Length do
+                            let c = source.[i-1]
+                            if c = '\r' && i < source.Length && source.[i] = '\n' then ()
+                            elif c = '\r' then yield i
+                            elif c = '\n' then yield i
                     |]
```
Comment (KevinRansom) [inline src/fsharp/NameResolution.fs]: @AviAvni.   What happens if source is a string with no '\r'  or '\n'.  I think it should yield source.Length.  However, this doesn't.  Or is that unnecessary?

Comment (AviAvni) [reply src/fsharp/NameResolution.fs]: It's yield source.Lemgth one line later

Comment (KevinRansom) [reply src/fsharp/NameResolution.fs]: Wierdly enough, it showed that line as deleted on my machine at work.  But yes, I see it now.

Thx.

**Comment by dsyme** [comment]
IIRC this code was added to FCS and there were normally tests added for everything we did there. So the FCS tests would be the right place

**Pull request merged**